etc/gcerc

```diff
@@ -124,6 +124,152 @@ echo \"run /var/lib/bootstrap-gce.sh\" ; \
   gssh $instname $cmd
 }
 
+gmaster() {
+  # build a master node of SOLVCON cluster.
+
+  usage="Usage: gmaster <instance_name>"
+
+  instname=$1
+  shift
+  if [ -z "$instname" ]; then
+    echo "Incorrect instance name"
+    echo $usage
+    return
+  fi
+
+  while [[ $# -gt 0 ]]
+  do
+  key="$1"
+
+  case $key in
+    -h|--help)
+    help="1"
+    ;;
+    *)
+      # unknown option
+    ;;
+  esac
+  shift # past argument or value
+  done
+
+  if [ -n "$help" ]; then
+    echo $usage
+    return
+  fi
+
+  script_location="$HOME/opt/gce/bin/admin/bootstrap-cluster.sh"
+  cmd="echo \"run $script_location master\" ; \
+$script_location master"
+  gssh $instname $cmd
+}
+
+gclient() {
+  # build a client node of SOLVCON cluster.
+
+  usage="Usage: gclient <instance_name> [-i master node IP]"
+
+  instname=$1
+  shift
+  if [ -z "$instname" ]; then
+    echo "Incorrect instance name"
+    echo $usage
+    return
+  fi
+
+  while [[ $# -gt 0 ]]
+  do
+  key="$1"
+
+  case $key in
+    -i|--ip-address)
+    ip_address="$2"
+    shift
+    ;;
+    -h|--help)
+    help="1"
+    ;;
+    *)
+      # unknown option
+    ;;
+  esac
+  shift # past argument or value
+  done
+
+  if [ -n "$help" ]; then
+    echo $usage
+    return
+  fi
+
+  script_location="$HOME/opt/gce/bin/admin/bootstrap-cluster.sh"
+  ip_address=`gce-guess-master`
```

Comment:
Where we use the helper script to guess the master IP address instead of pure hard-code.